router/pkg/config/config.schema.json

```diff
@@ -52,6 +52,24 @@
             }
           }
         },
+        "kv": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "required": ["url", "id"],
+            "additionalProperties": false,
+            "properties": {
+              "id": {
+                "type": "string",
+                "description": "The ID of the storage provider. The ID is used to identify the storage provider in the configuration."
+              },
+              "url": {
```

Comment:
Should we have additional fields for security or auth? Or is url enough to properly configure a Redis instance in production?

Reply:
I would assume just url to start with (with the auth string being included in the URL), but whatever you think makes sense

Reply:
I think users can configure an instance at this point with `redis://user:pass@localhost:6379`, and we can always add in additional fields for extra auth as customers request it

Reply:
Sounds good to me. URL was sufficient in case of Redis.